.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -146,4 +148,4 @@ kubectl run $POD_NAME \
       }
     ]
   }
-}' -- ./run "$COMMAND" $WORLD_PARAMETERS --parallel $PARALLEL_RUNS --retry $RETRIES --retry-tag-filter @Flaky --format junit:$JUNIT_REPORT_PATH
+}' -- ./run "$COMMAND" $WORLD_PARAMETERS --parallel $PARALLEL_RUNS --tags "not @PRA" --retry $RETRIES --retry-tag-filter @Flaky --format junit:$JUNIT_REPORT_PATH
```

Comment:
apart from the extra 3 "DR" parameters and the tags `@PRA` / `not @PRA`, this is the same script in both cases...
→ it probably would make more sense to keep a single script which gracefully ignores missing DR site, and takes a parameter with the tags (or just adds "$@" at the end of the run command, so that any extra param can be passed...)

```suggestion
}' -- ./run "$COMMAND" $WORLD_PARAMETERS --parallel $PARALLEL_RUNS --retry $RETRIES --retry-tag-filter @Flaky --format junit:$JUNIT_REPORT_PATH "$@"
```